hack/make/.build-rpm/docker-engine.spec

```diff
@@ -35,6 +35,7 @@ Requires(preun): initscripts
 
 # required packages on install
 Requires: /bin/sh
+Requires: docker-selinux
```

Comment:
I'm -1 on depending on some extra SELinux-specific package that we don't
build ourselves.  If there's something that "Docker on RedHat platforms"
needs, we should include it in either Docker, or in our packages (as
appropriate).